etc/qvm-start-daemon-kde.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=Qubes Guid/Pacat
+Comment=Starts GUI/AUDIO daemon for Qubes VMs in KDE
+Icon=qubes
+Exec=qvm-start-daemon --all --watch --kde
+Terminal=false
+Type=Application
+OnlyShowIn=KDE
```

Comment:
This needs to end with a semicolon: https://developer.gnome.org/menu-spec/#desktop-entry-extensions-examples
And also there is missing newline.